roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,104 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+class DockerImageAvailability(OpenShiftCheck):
+    """Check that required Docker images are available."""
+
+    name = "docker_image_availability"
+    tags = ["preflight"]
+
+    # attempt to pull required docker images and fail if
+    # any images are missing, or error occurs during pull
```

Comment:
This documentation comment would be best in the form of a docstring. I think it could complement the documentation of this class:

```python
class DockerImageAvailability(OpenShiftCheck):
    """Check that required Docker images are available.

    This check attempts to pull required docker images and fail if
    any image is missing or an error occurs during pull.
    """

```

For one, that's more idiomatic than having comments on top of a method definition. And while we don't do it today, there is tooling for extracting docstrings and generating documentation -- so, for example, we could generate a page documenting all existing checks in the future.